dip-0001.md

```diff
@@ -1,11 +1,113 @@
+
+### Ad integration
+
+Refers to the specific configuration or placement of advertisements. It outlines the mechanism through which ads are integrated into different platforms and mediums, such as newsletters, web applications, and mobile apps developed with React Native.
+
+### Variants
+
+Ad parameters and integrations slugs can be extended to provide additional specificity and flexibility. Variants are extensions denoted by a hyphen (-) followed by specific criteria, enabling to tailor ads more precisely according to size, aspect ratio, format, or protocol.
+
+## Specifications
+
+### `adParameters`
+
+Below is a detailed explanation of a first ad parameters list and its variants. Ad parameters are crucial for defining the properties and characteristics of advertisements.
+
+#### `imageURL`
+
+- Description: This field specifies the URL for the ad image, serving as a visual representation to capture viewers' attention.
+- Validation RegExp: `^\w+:\/\/[\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|\/))$`
+- Variants:
+  - Fixed Size: Specifies the required image fixed size. Refer to IAB's standard ad sizes.
+    - Exemple: `imageURL-320x50` indicates a fixed image size of 320x50 pixels, aligning with IAB's standard ad sizes.
+  - Image Ratio: Specifies the required image ratio for consistency across platforms.
+    - Exemple: `imageURL-1:1` specifies an image ratio of 1:1, ensuring consistency across different platforms.
+  - Protocol: Limits the image source to IPFS or HTTPS protocols.
+    - Exemple: `imageURL-ipfs` limits the image source to the IPFS protocol for secure and reliable delivery.
+  - Format: Constrains the image to specific formats for optimized loading.
+  - Protocol: Limits the image source to IPFS or HTTPS protocols.
+    - Exemple: `imageURL-jpg` restricts the image to the JPEG format for optimized loading times.
+
+#### `linkURL`
+
+- Description: Defines the destination URL to direct users upon ad interaction.
+- Validation RegExp: `^https:\/\/[^\[\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|\/))$`
+
+#### `xCreatorHandle`
+
+- Description: Identifies the X (Twitter) handle of the promoted individual or entity.
+- Validation RegExp: `^@[A-Za-z0-9_]{1,15}$`
+
+#### `xSpaceId`
+
+- Description: Indicates the unique ID for a promoted X (Twitter) Space.
+- Validation RegExp: `^\d[A-Za-z\d]{12}$`
+
+### `adIntegrations`
+
+Below is a detailed explanation of a first ad integrations list. Each integration is tailored for specific ad parameters and platforms.
+
+#### `BannerDynamic`
```

Comment:
DynamicBanner?

Reply:
I have not found a better name. I would mean it's a banner that selects ad data _randomly_. If you refresh the page integrating the banner, it may display something different. 
Any suggestion, an alternative name?

Reply:
Oh yes, I see the mistake. I will fix it in the next commit.